package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "todo",
```

Comment:
Add `author`, `license`, `repository`, `version`, `description`.
This can help: https://heynode.com/tutorial/what-packagejson/

Reply:
Sure
Thanks for your help